doc/notebooks/basic_example_grass_jupyter.ipynb

```diff
@@ -0,0 +1,179 @@
+    "The `grass.jupyter` package was written as part of Google Summer of Code in 2021. For more information, visit the [wiki page](https://trac.osgeo.org/grass/wiki/GSoC/2021/JupyterAndGRASS).\n",
+    "\n",
+    "\n",
+    "Examples here are using a sample GRASS GIS dataset for North Carolina, USA. The dataset is included in this environment.\n",
+    "\n",
+    "## Usage\n",
+    "\n",
+    "To run the selected part which is called a cell, hit `Shift + Enter`.\n",
+    "\n",
+    "## Start\n",
+    "\n",
+    "There are several ways to use GRASS GIS. When using Python in a notebook, we usually find GRASS GIS Python packages first, import them, initialize GRASS GIS session, and set several variables useful for using GRASS GIS in a notebook."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import Python standard library and IPython packages we need.\n",
+    "import os\n",
+    "import subprocess\n",
+    "import sys\n",
+    "\n",
+    "# Ask GRASS GIS where its Python packages are.\n",
+    "gisbase = subprocess.check_output([\"grass\", \"--config\", \"path\"], text=True).strip()\n",
+    "os.environ[\"GISBASE\"] = gisbase\n",
+    "sys.path.append(os.path.join(gisbase, \"etc\", \"python\"))\n",
+    "\n",
+    "# Import the GRASS GIS packages we need.\n",
+    "import grass.script as gs\n",
+    "import grass.jupyter as gj\n",
+    "\n",
+    "# Start GRASS Session\n",
+    "gj.init(\"../../data/grassdata\", \"nc_basic_spm_grass7\", \"user1\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Raster buffer\n",
+    "\n",
+    "Set computational region and create multiple buffers in given distances\n",
+    "around lakes represented as raster:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gs.parse_command('g.region', raster=\"lakes\", flags='pg')\n",
+    "gs.run_command('r.buffer', input=\"lakes\", output=\"lakes_buff\", distances=[60, 120, 240, 500])\n",
+    "\n",
+    "# Start a GrassRenderer map\n",
+    "r = gj.GrassRenderer()\n",
+    "\n",
+    "# Add a raster and vector to the map\n",
+    "r.d_rast(map=\"lakes_buff\")\n",
+    "r.d_legend(raster=\"lakes_buff\", range=(2, 5), at=(80, 100, 2, 10), flags=\"b\")\n",
+    "\n",
+    "# Display map\n",
+    "r.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Vector buffer\n",
+    "\n",
+    "Create a negative buffer around state boundary represented as a vector.\n",
+    "Vector modules typically don't follow computational region,\n",
+    "but we set it to inform display modules about our area of interest."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "gs.run_command('v.buffer', input=\"boundary_state\", output=\"buffer\", distance=-10000)\n",
```

Comment:
This comes from the original notebook, but it would be nice to have consistent quotes used everywhere (that is double quotes).